numpy/f2py/rules.py

```diff
@@ -202,7 +202,7 @@
 \tif (PyErr_Occurred())
 \t\t{PyErr_SetString(PyExc_ImportError, \"can't initialize module #modulename# (failed to import numpy)\"); return m;}
 \td = PyModule_GetDict(m);
-\ts = PyString_FromString(\"$R""" + """evision: $\");
+\ts = PyUnicode_FromString(\"$R""" + """evision: $\");
```

Comment:
The `$Revision: $` is a relict from subversion days and the line can be replaced with
```
\ts = PyUnicode_FromString(\"#f2py_version#\");
```
While doing that, I suggest inserting the line
```
f2py_version = numpy_version
```
in the header of the rules.py file because the f2py version is de facto defined by numpy version.